sources/common/toddl.go

```diff
@@ -145,13 +146,24 @@ func (ss *SchemaToSpannerImpl) SchemaToSpannerDDLHelper(conv *internal.Conv, tod
 			columnLevelIssues[srcColId] = issues
 		}
 
+		defaultVal := ddl.DefaultValue{
+			IsPresent: srcCol.DefaultValue.IsPresent,
+			Value:     srcCol.DefaultValue.Value,
```

Comment:
If the default value is spanner supported, then only we should populate the `Value` here right?

Reply:
updated